api.md

```diff
@@ -92,15 +117,23 @@ POST /api/v1/users/
 
 #### Response
 
-```
+```json
 {
+    "atat_permissions": [
+        "view_usage_report",
+        "view_usage_dollars",
+        "view_application_in_workspace",
+        "view_environment_in_application"
+    ],
     "atat_role": "developer",
-    "id": "4069cdec-6750-44d4-97d7-126658019040",
+    "id": "164497f6-c1ea-4f42-a5ef-101da278c012",
     "username": null,
     "workspace_roles": []
 }
 ```
 
+Returns a 201 if a new user was created, and a 200 if the user already exists.
```

Comment:
What's the motivation behind returning a 200 instead of a 409 when the user already exists?

Reply:
For now in atst, we're automatically creating new users in authz once they log in. This is made easier if POST /users is made more permissive (I would call it idempotent, but I don't think that it technically is). The distinction is that trying to create an already existing user is not an error, but a valid call.